test/Serialization.purs

```diff
@@ -43,6 +53,22 @@ suite :: TestPlanM (Aff Unit) Unit
 suite = do
   group "cardano-serialization-lib bindings" $ do
     group "conversion between types" $ do
+      test "PublicKey serialization" do
+        let
+          pkStr =
+            "ed25519_pk1p9sf9wz3t46u9ghht44203gerxt82kzqaqw74fqrmwjmdy8sjxmqknzq8j"
+          mPk = mkPubKey pkStr
+
+        pk <- liftM
+          (error $ "Failed to create PubKey from bech32string: " <> pkStr)
+          mPk
+
+        let
+          pkBytes = bytesFromPublicKey $ convertPubKey pk
+          (pk'' :: Maybe PublicKey) = mkFromCslPubKey <$> fromBytes
```

Comment:
The first set of parens are redundant

Reply:
@ngua Parsing fails without the parentheses.